test_app/models.py

```diff
@@ -19,8 +20,11 @@ class Organization(AbstractOrganization):
     pass
 
 
-class User(AbstractUser):
-    pass
+class User(AbstractUser, CommonModel):
+    # This doesn't get injected automatically by utils.models.decorate_user_model
+    # because we extend CommonModel which already inserts a summary_fields attribute.
+    def summary_fields(self):
+        return user_summary_fields(self)
```

Comment:
I already added logic to attach this method, I flied an issue for it and merged the fix which adds summary_fields when apps are ready

https://github.com/ansible/django-ansible-base/blob/devel/ansible_base/authentication/apps.py#L19

AWX will not have a custom user model. I tried very hard to get a solution, and it was a nightmare, and I don't have the time to complete that migration. The reason for dong the solution in apps ready methods is because that will work for services that don't have a custom model, which is going to be reality because that problem is nigh unfixable.